go-controller/pkg/ovn/base_network_controller_secondary.go

```diff
@@ -529,6 +508,31 @@ func (bsnc *BaseSecondaryNetworkController) removePodForSecondaryNetwork(pod *ka
 	return nil
 }
 
+// hasIPAMClaim determines whether a pod's IPAM is being handled by IPAMClaim CR.
+// pod passed should already be validated as having a network connection to nadName
```

Comment:
I love the function split here, its nice its split outside so the other function became simpler.